### PR TITLE
Swift documentation

### PR DESCRIPTION
**Pull request by bojan-txfusion**
Updated documentation after refactoring.

### CODE REVIEW

**Review by idea404 — APPROVED** [review]

File: docs/api/swift/paymaster-utils.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+| Parameter         | Type                  | Description                       |
+|-------------------|-----------------------|-----------------------------------|
+| `paymasterInput`  | `Data`                | The input data to the paymaster.  |
+
+```swift
+func encodeGeneral(_ paymasterInput: Data) -> Data
+```
+
+Check out the [example](getting-started.md#use-paymaster) how to use paymaster.
```
Comment (idea404) [inline docs/api/swift/paymaster-utils.md:50]: This is a broken link, mean to point somewhere else?

File: docs/api/swift/features.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## EIP-712 Metadata
+
+EIP712Meta contains EIP-712 transaction metadata.
+
+## Encoding paymaster params
+
+While the paymaster feature by itself does not impose any limitations on values of the `paymasterInput`, the Matter Labs team endorses certain types of [paymaster flows](../../reference/concepts/account-abstraction.md#built-in-paymaster-flows) that are processable by EOAs.
+
+zkSync SDK provides a utility method that can be used to get the correctly formed `PaymasterParams` object: [GetPaymasterParams](./paymaster-utils.md#getpaymasterparams).
```
Comment (idea404) [inline docs/api/swift/features.md:30]: This is also a broken link, someplace else?

File: docs/api/swift/accounts.md [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+
+It contains the same functions as [`WalletL1`](#walletl1), [`WalletL2`](#walletl2), and [`BaseDeployer`](#basedeployer) since it implements
+the Adapter interface, and the usage of those methods is the same.
+
+### `Init`
+
+Creates an instance of Wallet associated with the account provided by the signer. The `clientL2` and `clientL1` parameters are optional;
+if not provided, only `SignTransaction`, `Address` and `Signer` methods can be used, as the rest of the functionalities require
+communication with the network. A wallet that contains only a signer can be configured to communicate with L2 and L1 networks by using
+[`Connect`](#connect) and [`ConnectL1`](#connectl1), respectively.
```
Comment (idea404) [inline docs/api/swift/accounts.md:662]: These don't lead anywhere. Is this functionality still present?

File: docs/api/swift/accounts-l1-l2.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+method specification [`BaseCost`](accounts.md#basecost).
+
+## Claim failed deposit
+
+`WalletL1` and `Wallet` objects provide a claim fail deposit workflow. For more information, please refer to the method specification
+[`ClaimFailedDeposit`](accounts.md#claimfaileddeposit).
+
+## Finalize withdraw
+
+`WalletL1` and `Wallet` objects provide a finalize withdraw workflow. For more information, please refer to the method specification
```
Comment (idea404) [inline docs/api/swift/accounts-l1-l2.md:37]: Is this now only withdraw instead of finalize withdraw?

**Review by idea404 — CHANGES_REQUESTED** [review]

**Review by MexicanAce — APPROVED** [review]
LGTM (made a bunch of changes/fixes, it's not perfect but it's 99%)

**Pull request merged**